CMakeLists.txt

```diff
@@ -279,38 +279,38 @@ endfunction()
 # These options allow users to enable or disable the building of the various
 # protoc plugins. For example, running CMake with
 # -DgRPC_BUILD_GRPC_CSHARP_PLUGIN=OFF will disable building the C# plugin.
-set(_gRPC_PLUGIN_LIST)
+set(gRPC_PLUGIN_LIST)
```

Comment:
Why do we need this change?

Reply:
Names of variables used internally should start with an underscore. Because this variable is used inside the gRPCConfig.cmake.in and the name use the project namespace I have remove it.